django/template/engine.py

```diff
@@ -16,9 +16,10 @@ class Engine(object):
         'django.template.loader_tags',
     ]
 
-    def __init__(self, dirs=None, app_dirs=False, context_processors=None,
-                 debug=False, loaders=None, string_if_invalid='',
-                 file_charset='utf-8', libraries=None, builtins=None):
+    def __init__(self, dirs=None, app_dirs=False, autoescape=True,
```

Comment:
Since this initializer is a public API, I think we need to add new arguments at the end, for backwards compatibility.